src/pkgcheck/checks/cleanup.py

```diff
@@ -33,6 +33,17 @@ class RedundantVersionCheck(Check):
     _source = sources.PackageRepoSource
     known_results = frozenset([RedundantVersion])
 
+    @staticmethod
+    def mangle_argparser(parser):
+        parser.plugin.add_argument(
+            '--by-stable', action='store_true',
```

Comment:
Not sure, but maybe:
```suggestion
            '--stable-only', action='store_true',
```